onboarding_project/templates/admin/squad_pantry_app/order/change_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls %}
+{% block object-tools-items %}
+    <li>
+        <a href="{% url opts|admin_urlname:'history' original.pk|admin_urlquote %}" class="historylink">{% trans "History" %}</a>
+    </li>
+    {% if not request.user.is_kitchen_staff %}
+    <li>
+        <a href="cancel-order/" class="cancellink">{% trans "Cancel" %}</a>
+    </li>
+    {% endif %}
+{% endblock %}
```

Comment:
Newline required at the end of the file. Should have been caught in self-review.

Reply:
Done!